Documentation/Signoff/Logic-gate-subsystem.md

```diff
@@ -0,0 +1,100 @@
+# Logic Gates Subsystem
+
+## Function of the System
+The function of this subsystem is to introduce the concept of logic gates through an approachable way for someone with little to no experience using AND,OR and NOT gate. The subsystem will be user friendly by using switches to demonstrate these concepts and allow the user to interact with the inputs.
+
+
+## Constraint
+
+| No. | Constraints                                                           | Origin            |
+| --- | --------------------------------------------------------------------- | ----------------- |
+| 1   |  System shall operate at less than 50V, safe voltage level suitable for educational purposes.  | Safety Constraint |
```

Comment:
This is not a constraint on this system. I challenge you to find a mux that operates on more than 50 volts.